apps/condo/domains/resident/schema/RegisterResidentService.js

```diff
@@ -3,11 +3,18 @@
  */
 
 const { getById, GQLCustomSchema } = require('@core/keystone/schema')
-const access = require('../access/RegisterResidentService')
-const { Resident } = require('../utils/serverSchema/index')
-const { Property } = require('@condo/domains/property/utils/serverSchema')
-const { Resident: ResidentAPI } = require('../utils/serverSchema')
 
+const { Property: PropertyAPI } = require('@condo/domains/property/utils/serverSchema')
+const { Resident: ResidentAPI } = require('@condo/domains/resident/utils/serverSchema')
+const access = require('@condo/domains/resident/access/RegisterResidentService')
+
+/**
+ * Removes flat type and number from address string, if flatType provided and address contains it.
+ * @param address
+ * @param flatType
+ * @returns {*}
+ */
+const removeFlatNum = (address, flatType) => flatType ? address.split(`, ${flatType}`)[0] : address
```

Comment:
I've created a function getAddressUpToBuildingFrom with the same intent some time before.
Can it be reused in your cases?

Reply:
done